datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -769,7 +769,7 @@ impl DisplayAs for SortExec {
                         write!(
                             f,
                             // TODO should this say topk?
-                            "SortExec: fetch={fetch}, expr=[{}]",
+                            "SortExec: TopK(fetch={fetch}), expr=[{}]",
```

Comment:
If the sort does not use TopK, should we format it like `"SortExec: fetch={fetch}, expr=[{}]"`

Reply:
Yes, I think that is the intention